controller_manager/controller_manager/launch_utils.py

```diff
@@ -39,7 +39,8 @@ def generate_load_controller_launch_description(
         'joint_state_broadcaster',
         controller_type='joint_state_broadcaster/JointStateBroadcaster',
```

Comment:
should we deprecate this here also in #1502?

Reply:
Done in #1502